client/components/formatted-header/index.tsx

```diff
@@ -51,29 +48,22 @@ const FormattedHeader: FC< Props > = ( {
 		</InfoPopover>
 	);
 
-	const formattedHeaderText = disablePreventWidows ? headerText : preventWidows( headerText, 2 );
-	const formattedSubHeaderText = disablePreventWidows
-		? subHeaderText
-		: preventWidows( subHeaderText, 2 );
-
```

Comment:
The override fixed situations where the non-breaking space introduced by `preventWidows` would cause a horizontal scroll on smaller screens. `text-wrap: pretty` doesn't do that so is a safe replacement:

<img width="430" alt="Screenshot 2025-05-08 at 11 30 51 am" src="https://github.com/user-attachments/assets/ea864f18-a398-4235-8e3c-fbe01c502eef" />

Instance of `FormattedHeader` (search for a domain element):

<img width="764" alt="Screenshot 2025-05-08 at 11 42 10 am" src="https://github.com/user-attachments/assets/b0ad2b39-5034-4dc7-a4c5-44bd23e68ad4" />